src/DotVVM.Framework/ViewModel/Serialization/ViewModelSerializationMap.cs

```diff
@@ -399,6 +399,10 @@ public WriterDelegate CreateWriterFactory()
                     {
                         options["pathOnly"] = true;
                     }
+                    if (!property.TransferAfterPostback)
+                    {
+                        options["firstRequest"] = true;
```

Comment:
I'm not fan of transmitting all those options to the client. It's already annoyingly large.

Reply:
I know and we should definitely de-duplicate them and put them in the $type definitions together with the validation rules. But I'd rather solve this in a separate PR.